WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/eclipse/logic/event/listeners/BreakpointListener.java

```diff
@@ -37,8 +37,8 @@
 	private final Map<Integer, Breakpoint> breakpoints;
 
 	/** Constructor. */
-	public BreakpointListener(EventManager eventManager) {
-		this.eventManager = eventManager;
+	public BreakpointListener(DebugEventManager debugEventManager) {
+		this.debugEventManager = debugEventManager;
 		this.breakpoints = new HashMap<>();
```

Comment:
Oh right, I thought your comment was about another line, sorry. I'll update the type right now.